src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -19,8 +19,56 @@
          "groupId" : "com.google.cloud",
          "artifactId" : "google-cloud-bigquery",
          "version" : "0.22.0-beta"
+       }
       }
-    }
-   ]
+    ]
+  },
+  {
+    "name": "Google Cloud Datastore",
+    "documentation": "https://cloud.google.com/datastore/docs/reference/libraries",
+    "description":"Google Cloud Datastore is a NoSQL key-value database built for automatic scaling.",
+    "transport": "http",
```

Comment:
- Should `transport` be an array?  What do we do for libraries that support both `http` and `grpc`?
- Shouldn't this really be in the `clients` section?  We may have libraries that just use the `http` REST APIs, even though the API supports both

Reply:
do any libraries support more than one transport? let me check with the api team.

Reply:
still waiting for a reply but the docs do say, "Clients in this repository use either HTTP or gRPC for the transport layer. The README of each client documents the transport layer the client uses."

Reply:
Stepping back, what will we do with this information?  Are we going to use it as a filter — e.g., if `grpc` then cannot be used on AESv7?

Reply:
Yes, exactly that.

Reply:
So is better to encode this platform-compatibility knowledge into the consumers of this API, or to make it explicit in a compatibility/platform section?  I lean towards the latter.